interactions.go

```diff
@@ -63,17 +63,22 @@ type InteractionType uint8
 
 // Interaction types
 const (
-	InteractionPing = InteractionType(iota + 1)
+	InteractionPing InteractionType = iota + 1
 	InteractionApplicationCommand
```

Comment:
Also wanted to mention. I came across these constants and thought, maybe we should change the their names like so ```suggestion
	PingInteraction InteractionType = iota + 1
	ApplicationCommandInteraction
```
What do you think?

Reply:
What's the reasoning behind this change (especially with `InteractionButton` conforming to the old pattern)?

Reply:
Well, on my opinion, the new names looks and sounds better, plus this change will blend in perfectly with such enums as:

```go
const (
	// PrimaryButton is a button with blurple color.
	PrimaryButton ButtonStyle = iota + 1
	// SecondaryButton is a button with grey color.
	SecondaryButton
	// SuccessButton is a button with green color.
	SuccessButton
	// DangerButton is a button with red color.
	DangerButton
	// LinkButton is a special type of button which navigates to a URL. Has grey color.
	LinkButton
)
```

Reply:
I'm not sure I follow; it seems to me like the types of interactions should follow a consistent naming scheme, rather than all adhering to the buttons. I don't really see a reason to change this.